models/platform-limits/src/general/builder.spec.ts

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable jest/no-disabled-tests */
+/* eslint-disable jest/valid-title */
+import { createBuilderSpec } from '@commercetools-test-data/core/test-utils';
+import { TProjectCustomLimitsProjection } from './types';
+import * as ProjectCustomLimitsProjection from './index';
+
+describe('building', () => {
```

Comment:
These tests are testing the default generated model which has "null" as the value for all of its properties which does not provide too much value.
I believe it would help us better if we test the model created by the `withAllPlatformLimits` preset and check the values have at least a couple of the expected properties for each attribute.

Reply:
Done https://github.com/commercetools/test-data/pull/657/commits/6bfab6455c8b504b50e7e12d8b8c3e1da173dcef#diff-c5cb7d76dd373d32885396efb72c25d87c7d548a6671dba26273537a306cdfa4